build.sbt

```diff
@@ -1,27 +1,62 @@
+addCommandAlias("check", "; scalafmtSbtCheck; scalafmtCheckAll; compile:scalafix --check ; test:scalafix --check")
+addCommandAlias("fix", "; scalafmtSbt; scalafmtAll; compile:scalafix ; test:scalafix")
+
 ThisBuild / organization := "com.avast"
 ThisBuild / homepage := Some(url("https://github.com/avast/scala-server-toolkit"))
 ThisBuild / description := "Functional programming toolkit for building server applications in Scala."
 ThisBuild / licenses := Seq("MIT" -> url("https://raw.githubusercontent.com/avast/scala-server-toolkit/master/LICENSE"))
 ThisBuild / developers := List(Developer("jakubjanecek", "Jakub Janecek", "janecek@avast.com", url("https://www.avast.com")))
-ThisBuild / scmInfo := Some(
-  ScmInfo(url("https://github.com/avast/scala-server-toolkit"), "scm:git:git@github.com:avast/scala-server-toolkit.git")
-)
 
 ThisBuild / scalaVersion := "2.13.0"
-ThisBuild / scalacOptions := ScalacOptions.default
 
 ThisBuild / turbo := true
 
 lazy val commonSettings = Seq(
   libraryDependencies ++= Seq(
     compilerPlugin(Dependencies.kindProjector),
+    compilerPlugin(Dependencies.betterMonadicFor),
+    compilerPlugin(Dependencies.silencer),
+    compilerPlugin(scalafixSemanticdb), // for scalafix
+    Dependencies.silencerLib,
     Dependencies.catsEffect,
-    Dependencies.scalaTest
+    Dependencies.scalaTest % Test
+  ),
+  Compile / compile / wartremoverErrors ++= Warts.all filterNot Set(
```

Comment:
I think this should be refactored to some other file same way compiler options are refactored.

Reply:
ok, that is probably a good idea